packages/rulesets/src/spectral/test/get-collection-only-has-value-nextlink.test.ts

```diff
@@ -62,15 +193,21 @@ test("GetCollectionOnlyHasValueAndNextLink should find no errors when get collec
     },
   }
   return linter.run(myOpenApiDocument).then((results) => {
-    expect(results.length).toBe(0)
+    expect(results.length).toBe(1)
+    expect(results[0].path.join(".")).toBe(
+      "paths./subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/dnsZones/{zoneName}/dnssecConfigs.get.responses.200.schema.properties"
+    )
+    expect(results[0].message).toBe(
+      "Get endpoints for collections of resources must only have the `value` and `nextLink` properties in their model."
+    )
   })
 })
 
-test("GetCollectionOnlyHasValueAndNextLink should find errors when get collection schema has properties other than value and nextLink", () => {
+test("GetCollectionOnlyHasValueAndNextLink should find errors when get collection schema has two provider namespaces", () => {
   const myOpenApiDocument = {
     swagger: "2.0",
     paths: {
-      "/api/Paths": {
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/dnsZones/{zoneName}/dnssecConfigs": {
```

Comment:
>Microsoft.Network/dnsZones/{zoneName}/dnssecConfigs": [](http://example.com/codeflow?start=84&length=53)

I am seeing only Microsoft.Network here . which is the other namespace?

Reply:
Added